### PR TITLE
Tasks from issue #51 sprinkled in with QOL updates

### LINKED ISSUE #51: create CRUD backend for notifications
- [x] accept/deny requests
- [x] logout on profilepage
- [x] reload header to show loggedin or loggedout
- [x] number on product
- [x] show location on product
- [x] show rented on product

### PR DESCRIPTION
**Pull request by PettoDavida**

### CODE REVIEW

**Review by Vacannot — APPROVED** [review]
honestly i didn't look but LGTM

File: src/components/userProfile/UserInfoCard.tsx [hunk 1]
```diff
@@ -158,20 +160,32 @@ const View = ({user}: {user: any}) => {
...
+          }}
+        >
+          Delete
+        </Button>
+        <Button
+          variant="outlined"
+          endIcon={<LogoutOutlined />}
+          onClick={() => {
+            logout();
+            navigate("/");
```
Comment (Vacannot) [inline src/components/userProfile/UserInfoCard.tsx:183]: In my earlier version i needed to do window.location.reload() to actually update the header state after logging out, does this still have that behaviour?

**Review by PettoDavida — COMMENTED** [review]
yes

File: src/components/userProfile/UserInfoCard.tsx (hunk 1 above)
Comment (PettoDavida) [reply src/components/userProfile/UserInfoCard.tsx:183]: https://github.com/Vacannot/pinkrent/blob/a563dafb59255d037952527ba3612a9fb38f13ee/src/components/Header/Header.tsx#L25-L31

**Review by maxAndersson — APPROVED** [review]

**Review by fiababiakandersson — APPROVED** [review]

**Pull request merged**